tests/e2e/daemonset-features/scc.yaml

```diff
@@ -0,0 +1,22 @@
+kind: SecurityContextConstraints
```

Comment:
It's similar to what jaeger-operator does https://github.com/jaegertracing/jaeger-operator/blob/dcb7d7f4bec09ab74b17cac799c3502c6b314158/examples/openshift/hostport-scc-daemonset.yaml

Reply:
Yes. I copied, pasted and adapted.